### ISSUE TITLE
missing INCLUDE files in POLYMER-03.DATA

### ISSUE THREAD

**Issue by GitPaean** [OPM/opm-tests#962]
```
Error: No such file: opm-tests/polymer/include/POLYMER-03-POLYMER.inc
```

it was introduced in commit 646df94c2e0979a93daa41c3ce6caeae184622e0 . @OPMUSER , can you have a look?

**Comment by OPMUSER** [comment]
Just pushed a PR with the missing file.

**Comment by GitPaean** [comment]
fixed by #965
